File: main/01_initializing_data.R
## Creating a data set
# base and/or tidyverse
library(tidyverse)
df = data.frame('Name' = c('Alfred', 'Batman', 'Charlie', 'Delta', 'Echo'), 
                 'Age' = c(99, 50, 30, 20, 0), 
                 'Random' = c('Yes', 'No', 'Yes', 'No', 'No'), 
                 'Interesting' = c(TRUE, TRUE, FALSE, FALSE, TRUE))

# data.table
library(data.table)
dt = data.table('Name' = c('Alfred', 'Batman', 'Charlie', 'Delta', 'Echo'), 
            'Age' = c(99, 50, 30, 20, 0),
            'Random' = c('Yes', 'No', 'Yes', 'No', 'No'), 
            'Interesting' = c(TRUE, TRUE, FALSE, FALSE, TRUE))


## Reading in data
# base and/or tidyverse
library(tidyverse)
df = read.delim('00_R_data_set.tsv')

df = read_tsv('00_R_data_set.tsv')

# data.table
library(data.table)
dt = fread('00_R_data_set.tsv')


## Writing data
# base and/or tidyverse
library(tidyverse)
write.table(df, '00_R_data_set_new.tsv', sep='\t', quote=FALSE, row.names=FALSE)

write_tsv(df, '00_R_data_set_new.tsv')

# data.table
library(data.table)
fwrite(dt, '00_R_data_set_new.tsv', sep='\t')

File: main/01_initializing_data.jl
## Creating a data set
# DataFrames
using DataFrames
df = DataFrame(Name = ["Alfred", "Batman", "Charlie", "Delta", "Echo"], 
Age = [99, 50, 30, 20, 0], 
Random = ["Yes", "No", "Yes", "No", "No"], 
Interesting = [true, true, false, false, true])


## Reading in data
# CSV
using CSV
df = CSV.read("main/00_jl_data_set.tsv")


## Writing data
# CSV
using CSV
CSV.write("main/00_jl_data_set_new.tsv", df , delim="\t")